modules/actuators/servo.py

```diff
@@ -77,8 +85,13 @@ def execute_move(self, sequence):
         if self.power:
             pub.sendMessage('power:use')
         if self.serial:
-            # print(int(s[0]))
-            pub.sendMessage('serial', type=ArduinoSerial.DEVICE_SERVO, identifier=self.pin, message=s[0])
+            # just move the pan servo for now. Remove after debugging
+            if self.index != 7 and self.index != 6:
```

Comment:
This discards all commands that are not for the neck (legs). It can be removed but the pi  leg configuration has not been finalised.